src/omnipool/math.rs

```diff
@@ -321,6 +322,30 @@ pub fn calculate_remove_liquidity_state_changes(
         Balance::zero()
     };
 
+    // Calculate withdrawal fee
+    let price_diff = if oracle_price <= current_price {
```

Comment:
maybe use abs_diff instead?

https://doc.rust-lang.org/std/primitive.u128.html#method.abs_diff

Reply:
It is fixed u128. It does not have abs diff. Otherwise I would of course